gems/pending/appliance_console.rb

```diff
@@ -94,8 +96,24 @@
 require 'appliance_console/prompts'
 include ApplianceConsole::Prompts
 
+# Updates the ip address associated with a hostname in the /etc/hosts file
+#
+# @param host [String] The hostname to look for
+# @param ip [String] The address to write
+def update_hostname_ip(host, ip)
```

Comment:
This method should live in the ApplianceConsole namespace so as not to bleed into the toplevel namespace.